docs/imm-otel.md

```diff
@@ -0,0 +1,110 @@
+# Deploy the SignalFx Smart Agent and OpenTelemetry Collector for Infrastructure Monitoring
+
+SignalFx Infrastructure Monitoring helps you gather metrics related to your system's performance. Metrics provide information about processes running inside the system, including counters, cumulative counters, and gauges.
+
+The OpenTelemetry Collector is a stand-alone service that can ingest metrics from various sources. The OpenTelemetry collector provides additional flexibility in configuration options. You can now configure integrations to send data to an OpenTelemetry Collector to centrally manage data sent to SignalFx.
+
+This is two-step process:
+
+1. [Deploy the SignalFx Smart Agent](#deploy-the-signalfx-smart-agent)
+2. [Deploy an OpenTelemetry Collector for SignalFx](#deploy-an-opentelemetry-collector-for-signalfx)
+
+The following sections describe each step in detail.
+
+## Deploy the SignalFx Smart Agent
+
+The SignalFx platform is built to ingest, store, analyze, visualize, and alert on metrics at scale. A metric is anything that is measurable (you can assign a numerical value to it) and variable (changes over time). The following are examples of metrics:
+
+* CPU utilization % of a server
+* Response time in milliseconds of an API call
+* The number of unique users who logged in over the previous 24-hour period
+
+You can use the SignalFx Smart Agent to collect metrics for SignalFx Infrastructure Monitoring. See [Use the Smart Agent](https://docs.signalfx.com/en/latest/integrations/agent/index.html#smart-agent "Use the Smart Agent") for more information.
+
+**Step 1: Install the SignalFx Smart Agent**
+
+There are two options for installing the SignalFx Smart Agent:
+- Use the [quick installation](https://docs.signalfx.com/en/latest/integrations/agent/quick-install.html "Quick Install") option for simplified SignalFx Smart Agent command-line installation on a single host.
+- Use the [advanced installation](https://docs.signalfx.com/en/latest/integrations/agent/advanced-install-options.html "Advanced Installation Options") option for bulk deployments and for configuring various monitors for your environment.
+
+**Step 2: Configure the SignalFx Smart Agent**
```

Comment:
we need to address what it looks like to send data to an otel collector, e.g., including the hostname:port of the otel collector to export data to the collector. 

even though we link to an example config on line 34, we could link to this for additional example configs: https://github.com/signalfx/signalfx-agent/tree/master/deployments.